client/.eslintrc.js

```diff
@@ -1,15 +1,14 @@
 module.exports = {
 	rules: {
-		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
+		'import/no-extraneous-dependencies': [
+			'error',
+			{ packageDir: [ __dirname, __dirname + '/..' ] },
```

Comment:
This tells `eslint` that is ok to do import from `wp-calypso`, as `wp-calypso` is a package defined in `../package.json`